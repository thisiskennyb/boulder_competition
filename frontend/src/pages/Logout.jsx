import { Logout as fetchFunc } from "../authapi/auth";
import { toast } from "react-toastify";
import { useEffect } from "react";

export default function Logout() {
    useEffect(() => {
        fetchFunc()
    }, [])
    


    return (
        <>
        <h1 className="text-xl5 text-rose-400">Thanks for stopping by!!</h1>
        </>
    )
}