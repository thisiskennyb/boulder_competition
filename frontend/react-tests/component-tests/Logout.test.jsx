import {expect, it} from "vitest"
import Logout from "../../src/pages/Logout"

it("Logout renders without crashing", ()=> {
    expect(<Logout/>).toMatchSnapshot()
})