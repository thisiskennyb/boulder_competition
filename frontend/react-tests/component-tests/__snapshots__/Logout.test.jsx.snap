// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Logout renders without crashing 1`] = `<Logout />`;
